src/ext_deps/mjson.c

```diff
@@ -450,7 +450,7 @@ static int json_internal_read_object(const char *cp,
 		    if (!decimal && (seeking == t_integer || seeking == t_uinteger))
 			break;
 		}
-		if (cursor[1].attribute==NULL)	/* out of possiblities */
+		if (cursor[1].attribute==NULL)	/* out of possibilities */
```

Comment:
mjson is a external dependency, these should not be edited since they are maintained elsewhere.
This is for all files in ext_deps